src/lib/OpenEXR/ImfStandardAttributes.h

```diff
@@ -299,17 +762,103 @@ IMF_STD_ATTRIBUTE_DEF (framesPerSecond, FramesPerSecond, Rational)
 
 
 //
-// multiView -- defines the view names for multi-view image files.
+// multiView -- defines the view names for multi-view image files
+//
 // A multi-view image contains two or more views of the same scene,
 // as seen from different viewpoints, for example a left-eye and
 // a right-eye view for stereo displays.  The multiView attribute
 // lists the names of the views in an image, and a naming convention
 // identifies the channels that belong to each view.
 //
+// Files conformant to SMPTE ST 2065-4 ("ACES Container File Format")
+// which have a multiView attribute will always have the two strings
+// "right" and "left" in that order. When an ACES Container file is
+// used to hold a stereoscopic image, this attribute will be present.
+//
 // For details, please see header file ImfMultiView.h
 //
 
-IMF_STD_ATTRIBUTE_DEF (multiView , MultiView, StringVector)
+IMF_STD_ATTRIBUTE_DEF (multiView, MultiView, StringVector)
+
+
+//
+// framelines -- set of rectangles used for creative purposes, e.g. to
+// indicate safe areas, cropping for derivative image sequences, etc
+//
+// The way in which the format encodes these rectangles is not defined;
+// the encoding might be camera-vendor-specific, or it might be formally
+// or informally standardized (e.g. an ASC Frameline Decision List)
+//
+
+IMF_STD_ATTRIBUTE_DEF (framelines, Framelines, std::string)
+
+
+//
+// imageRotation -- rotation of the image in degrees, in the closed interval
+// [-45, +45], with the rotation around the center of the display window, a
+// positive value specifying a rotation from x-axis to y-axis
+//
+
+IMF_STD_ATTRIBUTE_DEF (imageRotation, ImageRotation, float)
+
+
+//
+// cameraPosition -- x, y, z position of the camera, in meters
+//
+// The attribute may record the center of the camera's sensor, or the
+// center of the entrance pupil of the lens. The coordinate system for
+// the camera's position is fixed in relation to the set, and is a
+// right-handed Cartesian coordinate system with the z-axis pointing
+// upwards and the y-axis pointing 90 degrees to the left of the
+// x-axis direction.
+//
+// Recording the position of the center of the entrance pupil provides the
+// most usable value in some circumstances, e.g. as the camera pans. This
+// position changes, however, with lens and focus. For cases when this
+// precision is not needed, recording the center of the sensor is provided
+// as an alternative.
+//
+// For higher precision the chosen coordinate system is usually NOT the GPS
+// system, but an on-set coordinate system. Any coordinate system can be 
+// chosen for recording the camera's position, as long as its axes are
+// orthogonal, and z points up. The coordinate system can be moving with
+// the set, for example, when on a ship. Its reference point (usually
+// assignged position 0,0,0) can be within or outside the set.
```

Comment:
typo assignged

Reply:
Fixed, thanks.

Reply:
If you push a commit to your JGoldstone:develop branch, it will automatically appear here as part of the conversation flow. That's a clever bit about the way the GitHub workflow operates. The next step is I can hit "Resolve Conversation", and this small bit of the review flow will fold down to indicate resolution.